adapter/avro/src/Mu/Adapter/Avro/Example.hs

```diff
@@ -35,4 +36,8 @@ type Example = [avro|
 }
 |]
 
+#if __GHCIDE__
```

Comment:
Can we somehow make all paths to files work for ghcide/cabal/stack/nix, etc... by tweaking somehow the `*.cabal` file? 🤔 Maybe that way we can skip using `CPP` all over the examples.

Reply:
I don't know how to do it without using even more CPP magic :(